tikv_metrics_proxy/README.md

```diff
@@ -0,0 +1,49 @@
+TiKV metrics proxy
+------
+
+TiKV metrics proxy is a proxy for the TiKV server's prometheus metrics. The `tikv_metrics_proxy` exposes metrics via HTTP on the `/metrics` endpoint. 
+
+#### Build
+
+```
+make tikv_metrics_proxy
+```
+
+#### Running
+
+```
+$ ./tikv_metrics_proxy -h
+usage: tikv_metrics_proxy [<flags>]
+
+Flags:
+  -h, --help              Show context-sensitive help (also try --help-long and --help-man).
```

Comment:
What are `help-long` and `help-man`?

Reply:
It's build-in help message in kingpin package:
https://github.com/alecthomas/kingpin/blob/7e2d1284f714168af95785926b6c7da2aa0abd21/app.go#L62